server/core/src/main/java/org/infinispan/server/core/backup/BackupManagerImpl.java

```diff
@@ -43,14 +45,17 @@
    final BackupReader reader;
    final Lock backupLock;
    final Lock restoreLock;
+   private final EmbeddedCacheManager cacheManager;
    final Map<String, DefaultCacheManager> cacheManagers;
    final Map<String, BackupRequest> backupMap;
    final Map<String, CompletionStage<Void>> restoreMap;
 
+
    public BackupManagerImpl(BlockingManager blockingManager, EmbeddedCacheManager cm,
                             Map<String, DefaultCacheManager> cacheManagers, Path dataRoot) {
       this.blockingManager = blockingManager;
       this.rootDir = dataRoot.resolve(WORKING_DIR);
+      this.cacheManager = cm;
```

Comment:
can't you use one of the `DefaultCacheManager` from `this.cacheManagers`?

Reply:
I have another PR to remove multiple cache managers that I'll refresh. This makes things simpler for now here.